### ISSUE TITLE
Multiple-hop Jumphost support (ssh -J)

### ISSUE THREAD

**Issue by simon202011** [blinksh/blink#1168]
The ProxyCommand support in Blink shell enables single-hop jumphost.
However, Blink doesn't seem to support nested ProxyCommand, eg:

ssh  -oProxyCommand='ssh -W %h:%p -oProxyCommand="ssh -W %h:%p -p 2222 user@host1"  -p 2222 user@host2' -p 2222  user@host3

In some environments, more than one hop is required in order to safely log into the destination server.

Please add support for the "ssh -J" option, enabling multiple-hop jumphost feature, eg:
ssh  -J user@jumphost1:2222,user@jumphost2:2222,user@jumphost3:2222    destination 

Thank you.

**Comment by carloscabanero** [comment]
This is already done in Blink 14. Now, for the current Blink, I am not sure if there is anything preventing the command from passing the rest of the CLI, but I would be surprised if it did. Are you sure the proxy on destination is not receiving the rest of the command?

**Comment by simon202011** [comment]
Which is included in Blink 14,  the -J option or nested ProxyCommand?  When do you estimate Blink 14 will be released to AppStore?  

Thank so much for your wonderful work.

**Comment by carloscabanero** [comment]
I can confirm that nested ProxyCommand works.

It could be an issue with the parsing of the command. The quotes must be around the whole option, so 
ssh -o "ProxyCommand=[...]"

This is anyway greatly simplified with -J. We will open testing in the next days.

**Comment by carloscabanero** [comment]
This is now available on TF 13.90.

You can now try this out by setting one or multiple JumpHosts following the expected OpenSSH syntax (https://www.cyberciti.biz/faq/linux-unix-ssh-proxycommand-passing-through-one-host-gateway-server/). Some examples:

```
ssh -J jumphost finalhost
ssh -J jumphostOne:port,jumphostTwo finalhost
```

Internally this is equivalent to using the old syntax with ProxyCommand:
```ssh -o ProxyCommand="ssh -W %h:%p jumphost” finalhost```

ProxyCommand is still available if someone has a more exotic scenario, like doing a nc on the remote or anything else.

Additionally, parameters on proxycommand do not require to also quote the “ProxyCommand=“ part anymore, as I posted before.
